lib/ai_helper/assistant.rb

```diff
@@ -3,35 +3,40 @@
 module DiscourseAi
   module AiHelper
     class Assistant
-      def available_prompts(name_filter: nil)
-        cp = CompletionPrompt
-        prompts = []
+      AI_HELPER_PROMPTS_CACHE_KEY = "ai_helper_prompts"
+
+      def available_prompts
+        prompts = Discourse.cache.read(AI_HELPER_PROMPTS_CACHE_KEY)
```

Comment:
This pattern of `return if it's on cache, otherwise fetch it, is already implemented by Discourse.cache.fetch